docs/UPGRADING.rst

```diff
@@ -34,3 +35,38 @@ This file describes the procedure for upgrading HIL to a new version.
 
      $ systemctl start httpd
      $ systemctl start hil_network
+
+Release Notes
+-------------
+
+0.3 (Upcoming)
+++++++++++++++
+
+* Validation checks for the configuration file have been added; be advised
+  that if there were errors in your ``hil.cfg`` that were previously
+  undetected you may need to fix them before the servers will start again
+* The interface to the ``hil`` command line tool has changed substantially;
```

Comment:
maybe we need to have an updated doc so we can point the readers to the relevant documentation?

Reply:
I think just mentioning `--help` should be enough. We could probably, in the future, add some documentation about the CLI (even a man page for HIL `man hil`).